lesson_14/exceptions/exceptions_app/src/main/java/com/codedifferently/lesson14/ecommerce/EcommerceSystem.java

```diff
@@ -17,19 +17,29 @@ public void addProduct(String productId, String name) {
     products.put(productId, new Product(productId, name));
   }
 
-  public String placeOrder(String productId, int quantity) {
+  public String placeOrder(String productId, int quantity) throws ProductNotFoundException {
     Product product = products.get(productId);
+    if (product == null) {
+      throw new ProductNotFoundException("Product with ID 1 not found ");
```

Comment:
This caused an error saying 'Expected "Product with ID 1 not found" but was "Product with ID 1 not found"